.github/workflows/airbyte-ci-tests.yml

```diff
@@ -13,15 +13,54 @@ on:
       - synchronize
     paths:
       - airbyte-ci/**
+    paths-ignore:
+      - '**/*.md'
 jobs:
   run-airbyte-ci-tests:
     name: Run Airbyte CI tests
     runs-on: "conn-prod-xlarge-runner"
     steps:
       - name: Checkout Airbyte
         uses: actions/checkout@v3
+      - name: Get changed files
+        uses: dorny/paths-filter@v2
+        id: changes
+        with:
+          filters: |
+            ops:
+              - 'airbyte-ci/connectors/connector_ops/**'
+            base_images:
+              - 'airbyte-ci/connectors/connector_ops/**'
+              - 'airbyte-ci/connectors/base_images/**'
+            pipelines:
+              - 'airbyte-ci/connectors/connector_ops/**'
+              - 'airbyte-ci/connectors/base_images/**'
+              - 'airbyte-ci/connectors/pipelines/**'
+            metadata-lib:
+              - 'airbyte-ci/connectors/connector_ops/**'
+              - 'airbyte-ci/connectors/metadata/lib/**'
+            metadata-orchestrator:
+              - 'airbyte-ci/connectors/connector_ops/**'
+              - 'airbyte-ci/connectors/metadata/lib/**'
+              - 'airbyte-ci/connectors/metadata/orchestrator/**'
```

Comment:
assuming the repetition of e.g. `connector-ops` path is due to internal lib dependencies?

Reply:
That was the idea! But I was wrong about metadata lib having a dependency on connector ops. Will update.